From e351f233334cb5484309133789646e165f4e64db Mon Sep 17 00:00:00 2001
From: Natanael Copa <natanael.copa@gmail.com>
Date: Wed, 11 Aug 2010 11:57:46 +0000
Subject: [PATCH] netdb: increase line size for /etc/services

If /etc/services has lines longer than 80 will getservbyname() fail
so we set it up to 160.

Signed-off-by: Natanael Copa <natanael.copa@gmail.com>
---
 libc/inet/getservice.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libc/inet/getservice.c b/libc/inet/getservice.c
index 03f5c29..8248f7a 100644
--- a/libc/inet/getservice.c
+++ b/libc/inet/getservice.c
@@ -29,7 +29,7 @@ aliases: case sensitive optional space or tab separated list of other names
 __UCLIBC_MUTEX_STATIC(mylock, PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP);
 
 #define	MAXALIASES	35
-#define BUFSZ		(80) /* one line */
+#define BUFSZ		(160) /* one line */
 #define SBUFSIZE	(BUFSZ + 1 + (sizeof(char *) * MAXALIASES))
 
 static parser_t *servp = NULL;
-- 
1.7.2.1

